packages/progress/progress.0.1.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "User-definable progress bars"
+description: """
+A progress bar library for OCaml, featuring a DSL for declaratively specifying
+progress bar formats. Supports rendering multiple progress bars simultaneously."""
+maintainer: ["Craig Ferguson <me@craigfe.io>"]
+authors: ["Craig Ferguson <me@craigfe.io>"]
+homepage: "https://github.com/CraigFe/progress"
+doc: "https://CraigFe.github.io/progress/"
+bug-reports: "https://github.com/CraigFe/progress/issues"
+depends: [
+  "dune" {>= "2.7"}
```

Comment:
Your `dune-project` file has `(lang dune 2.7.1)`. Should it require `2.7.1` instead? @rgrinberg

Reply:
mh, the `.1` part seems to be ignored by dune. Good to go then.